lib/livebook_web/live/home_live.ex

```diff
@@ -157,18 +160,40 @@ defmodule LivebookWeb.HomeLive do
     end
   end
 
+  defp update_notification(%{version: nil} = assigns), do: ~H""
+
+  defp update_notification(assigns) do
+    ~H"""
+    <div class="px-2 py-2 bg-blue-200 text-gray-900 text-sm text-center">
+      <span>
+        Livebook v<%= @version %> available! Check out the news on
+        <a class="font-medium border-b border-gray-900 hover:border-transparent" href="https://livebook.dev/" target="_blank">
+          livebook.dev
+        </a>
+        <%= if @instructions_url do %>
+          and follow the
+          <a class="font-medium border-b border-gray-900 hover:border-transparent" href={@instructions_url} target="_blank">
+            update instructions
+          </a>
+        <% end %>
```

Comment:
```suggestion
        Livebook v<%= @version %> available!
        <%= if @instructions_url do %>
          Check out the news on
          <a class="font-medium border-b border-gray-900 hover:border-transparent" href="https://livebook.dev/" target="_blank">
            livebook.dev
          </a>
          and follow the
          <a class="font-medium border-b border-gray-900 hover:border-transparent" href={@instructions_url} target="_blank">
            update instructions
          </a>
        <% else %>
          Check out the news and installation steps on
          <a class="font-medium border-b border-gray-900 hover:border-transparent" href="https://livebook.dev/" target="_blank">
            livebook.dev
          </a>
        <% end %>
```

I am slightly changing the text of the default case because reading it in practice made it feel weird. :D we are telling them to read stuff but reading stuff won't make the banner disappear. So I made it a bit clearer.